met-web/src/routes/LandingPage.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import EnhancedTable from "../components/layout/Table";
+import styled from "@emotion/styled";
+import Button from "@mui/material/Button";
+import sx from "mui-sx";
+import TemporaryDrawer from "../components/layout/Drawer";
+
+const PageContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const DrawerContainer = styled.div`
+  display: flex;
+  flex: 0.1;
+  height: 90vh;
+  align-items: center;
+  justify-content: center;
+`;
+
+const TableContainer = styled.div`
```

Comment:
We can consider Material UI <Grid> here to avoid extra css and multiple flex containers